src/Draggable.js

```diff
@@ -10,7 +10,7 @@ class Draggable extends React.Component {
     onDragStart: () => {},
     onDrag: () => {},
     data: null,
-    type: null,
+    type: '',
```

Comment:
I don't know about this change here, I think it makes more sense to send a a null if nothing is being draggged. It would be more consistent with the other properties of the state.
Is this change intended to add better support for flow by not switching between null and a string?

Reply:
Nope, it's about behavior.
> fixed default Draggable type which previously produced isOverAccepted: true when dragging cursor is not over Droppable and its accepts prop is string

Reply:
We need some more valuable default than `null`. Or we can make it required instead

Reply:
Also we can make `accepts` always array then `null` won't be found in it.

Reply:
I got an idea. We can you array as `accepts` default instead of `null`. It shouldn't be a breaking change.

Reply:
I hope I'm understanding correctly. I think it would make the most sense to have `isOverAccepted` evaluate to `true` if the `accepts` prop of the droppable is either null or undefined. That way you can default to accepting everything. The docs should probably be updated to make that clear.

Reply:
But the problem is that `isOverAccepted: true` even before hovering droppable. It's just always true.

Reply:
Oh I see what you're saying. So it evaluates to true if they're both null right? I guess in that case we could a truthy check on the currently hovered droppable id like:
```
isOverAccepted: currentlyHoveredDroppableId
    ? Array.isArray(this.currentlyHoveredDroppableAccepts)
        ? this.currentlyHoveredDroppableAccepts.includes(state.type)
            : state.type === this.currentlyHoveredDroppableAccepts
        : false
```

Reply:
Would that resolve what you're seeing?

Reply:
I will add test case for default